scikit-hep/environment.yml

```diff
@@ -0,0 +1,46 @@
+name: eschool2021_shep
```

Comment:
Shall one remove e.g. all the astro stuff as I won't be needing it?

Reply:
Yes sure, you may remove what is not needed there.

Reply:
I will then remove unnecessary stuff when commiting a basic README, etc., so that we can proceed here. Thanks.